packages/ckeditor5-dev-ci/bin/circle-workflow-notifier.js

```diff
@@ -67,7 +67,19 @@ const {
 	CIRCLE_JOB
 } = process.env;
 
-const { task, ignore } = parseArguments( process.argv.slice( 2 ) );
+const { values: { task, ignore } } = parseArgs( {
+	options: {
+		task: {
+			type: 'string',
+			default: 'pnpm ckeditor5-dev-ci-notify-circle-status'
+		},
+		ignore: {
+			type: 'string',
+			multiple: true,
+			default: []
+		}
+	}
+} );
```

Comment:
Technically, this is a breaking change, since the old implementation accepted arguments without the `=` sign (e.g., `--arg value`), while the new one only accepts arguments with it (e.g., `--arg=value`).

However, I looked at all usages of this command in `ckeditor` and `cksource` orgs, and none of them passed any parameters.